presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -164,6 +164,22 @@ public void testSTIsEmpty()
         assertFunction("ST_IsEmpty(ST_GeometryFromText('POLYGON EMPTY'))", BOOLEAN, true);
     }
 
+    @Test
+    public void testSTIsSimple()
+    {
+        assertFunction("ST_IsSimple(ST_GeometryFromText('POLYGON EMPTY'))", BOOLEAN, true);
```

Comment:
To shorten this test, consider introducing helper functions: assertSimpleGeometry and assertNotSimpleGeometry.

assertSimpleGeometry("POINT EMPTY");
assertNotSimpleGeometry("LINESTRING (0 0, 1 1, 1 0, 0 1)");

For clarity, consider grouping tests by geometry type. E.g. first test points, then multi-points, then linestrings, etc.